portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,28 @@
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  width: 1000px;
```

Comment:
Avoid using a fixed value. Consider using percentage based value, or CSS media queries to make sure the experience is good in any size of windows.